debezium-core/src/main/java/io/debezium/heartbeat/HeartbeatImpl.java

```diff
@@ -42,7 +46,11 @@
      */
     static final String DEFAULT_HEARTBEAT_TOPICS_PREFIX = "__debezium-heartbeat";
 
-    private static Schema KEY_SCHEMA = Schema.STRING_SCHEMA;
+    private static final String SERVER_NAME_KEY = "name";
```

Comment:
Let's name it "serverName", to make it a bit more contextual. This matches the name of the corresponding connector option (`database.server.name`).